renderapi/render.py

```diff
@@ -105,36 +143,38 @@ def connect(host=None, port=None, owner=None, project=None,
             client_scripts=None, client_script=None, memGB=None,
             force_http=True, validate_client=True, web_only=False, **kwargs):
     '''
-    helper function to connect to a render instance
-        can default to using environment variables if not specified in call.
-    input:
-        host -- string hostname for target render server -- will prepend
+    helper function to create a :class:`RenderClient` instance.
```

Comment:
This generates a `renderapi.render.Render` instance, and a client instance if it passes validation.